src/main/scala/tech/cryptonomic/conseil/routes/PlatformDiscovery.scala

```diff
@@ -12,7 +17,8 @@ import scala.concurrent.ExecutionContext
 
 /** Companion object providing apply implementation */
 object PlatformDiscovery {
-  def apply(config:  PlatformsConfiguration)(implicit apiExecutionContext: ExecutionContext): PlatformDiscovery = new PlatformDiscovery(config)
+  def apply(config:  PlatformsConfiguration)(implicit apiExecutionContext: ExecutionContext, system: ActorSystem): PlatformDiscovery =
```

Comment:
Is it still useful or meaningful to keep the ExecutionContext implicit, here and in the other routes classes?

Reply:
not useful when I've added